### ISSUE TITLE
Email sending breaks in python 3.x?

### ISSUE THREAD

**Issue by mflaxman** [themartorana/python-postmark#42]
I migrated to python 3 and this code is now breaking:

```
/Users/lemon/workspace/projectb/emails/trigger.py in postmark_send(subject, html_body, to_info, from_info, cc_info, replyto_info, bcc_info)
     22             reply_to=replyto_info,
     23             )
---> 24     return pm.send()
     25 
     26 

/usr/local/lib/python3.3/site-packages/postmark/core.py in send(self, test)
    460             result.close()
    461             if result.code == 200:
--> 462                 self.message_id = json.loads(jsontxt).get('MessageID', None)
    463                 return True
    464             else:

/usr/local/Cellar/python3/3.3.5/Frameworks/Python.framework/Versions/3.3/lib/python3.3/json/__init__.py in loads(s, encoding, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)
    314             parse_int is None and parse_float is None and
    315             parse_constant is None and object_pairs_hook is None and not kw):
--> 316         return _default_decoder.decode(s)
    317     if cls is None:
    318         cls = JSONDecoder

/usr/local/Cellar/python3/3.3.5/Frameworks/Python.framework/Versions/3.3/lib/python3.3/json/decoder.py in decode(self, s, _w)
    349 
    350         """
--> 351         obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    352         end = _w(s, end).end()
    353         if end != len(s):

TypeError: can't use a string pattern on a bytes-like object
```

I believe `jsontxt` needs a `.decode()`

The message sends, but the error would have you believe otherwise (and messes up your application logic).

**Comment by pingometer** [comment]
Also, looks good here in Python 3.4.2.

**Comment by themartorana** [comment]
Sorry for the delay. Fixed, tagged 0.4.5, uploaded to PyPi.
